app/src/test/java/org/simple/clinic/home/patients/PatientsScreenControllerTest.kt

```diff
@@ -43,6 +45,8 @@ class PatientsScreenControllerTest {
 
   @Before
   fun setUp() {
+    RxJavaPlugins.setIoSchedulerHandler { Schedulers.trampoline() }
```

Comment:
@vinaysshenoy can you add a comment here why this is being used?

Reply:
Done.